addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -242,6 +255,28 @@ def set_custom_notifications(self, custom_notifications):
         self.custom_notifications = custom_notifications
         self._bus_send_store(self.channel_id, {"custom_notifications": self.custom_notifications})
 
+    @api.ondelete(at_uninstall=False)
+    def _unlink_transfer_channel_owner(self):
+        for channel_member in self:
```

Comment:
I can't seem to be able to trigger this, what is the flow?

Reply:
when you are the chnanle owner and you are leaving the channel, you will get this.